src/irmin/path.ml

```diff
@@ -54,3 +54,13 @@ module String_list = struct
 
   let t = Type.like ~cli:(pp, of_string) Type.(list step_t)
 end
+
+module V1 = struct
```

Comment:
Unrelated change?

Could do with:
- [ ] a CHANGES entry
- [ ] (perhaps) a test
- [ ] (perhaps) a doc-string saying what makes this `String_list` implementation different

Reply:
I'll remove this, this is indeed not needed.